rero_mef/all_mef.py

```
@@ -0,0 +1,24 @@
+
+
+class AllMefSearch(RecordsSearch):
+    """RecordsSearch over the all_mef alias."""
+
+    class Meta:
+        """Search only on all_mef alias."""
+
+        index = "all_mef"
+        doc_types = None
+        fields = ("*",)
+        facets = {}
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)